src/leodb/db.cpp

```diff
@@ -12,6 +14,7 @@ bool DB<T, U>::put(Key<T> key, Value<U> value) {
     try {
         int inserted = key.hashItem();
         table[inserted] = Entry<T, U>(key, value);
+        totalKeys += 1;
```

Comment:
duplicate entries shouldn't increment the count

Reply:
Yup added: 
```
if (table.count(inserted) == 0) {
        totalKeys += 1;
    }
```